README.md

```diff
@@ -18,18 +18,32 @@ To set the development environment consider the following commands:
 $ cd obs-service-replace_using_package_version
 
 # Initiate the python3 virtualenv
-$ python3 -m venv .env3
+$ poetry shell
 
-# Activate the virutalenv
-$ source .env3/bin/activate
-
-# Install development dependencies
-$ pip install -r requirements.txt
+# Install the project and dependencies into the virtual env
+$ poetry install
 
 # Run tests and code style checks
 $ tox
 ```
 
+In case the development host is on a distro which does not include poetry
+one can still manually create a virtual environment and install poetry there.
+
+```bash
+# Create a virtual env
+$ python3 -v env .env3
+
+# Load the venv
+$ source .env3/bin/activate
+
+# Install poetry
+$ pip install poetry
+
+# Install the project and dependencies to the virtual env
+$ poetry install
+```
```

Comment:
Installing poetry into a virtualenv sounds like a bad idea to me to be honest (poetry will install packages into a global folder, but now you have a local poetry instance…). I think we should just link to the upstream installation guide: https://python-poetry.org/docs/#installation

Reply:
But yes probably is just better to link to https://python-poetry.org/docs/#installation so we do not have to deal with it at all. After all installing poetry is not that hard and even googling anyone will finds tons of blog posts, docs, forums, etc.